api/src/main/java/org/pmiops/workbench/api/CloudTaskInitialCreditsExpiryController.java

```diff
@@ -233,7 +232,7 @@ private Map<Long, DbUser> findDbUsersWithChangedCosts(
   private Set<DbUser> getFreeTierActiveWorkspaceCreatorsIn(Set<DbUser> users) {
     return workspaceDao.findCreatorsByBillingStatusAndBillingAccountNameIn(
         BillingStatus.ACTIVE,
-        new ArrayList<>(workbenchConfig.get().billing.freeTierBillingAccountNames()),
+        List.of(workbenchConfig.get().billing.initialCreditsBillingAccountName()),
```

Comment:
there has only been one initial-credits billing account for a long time, so I removed the plural logic